kernel_gateway/gatewayapp.py

```diff
@@ -144,6 +144,13 @@ def _default_kernel_name_default(self):
         # defaults to Jupyter's default kernel name on empty string
         return os.getenv(self.default_kernel_name_env, '')
 
+    list_kernels_env = 'KG_LIST_KERNELS'
+    list_kernels = Bool(config=True,
+        help='Controls whether /api/kernels will report running kernels and their IDs (KG_LIST_KERNELS env var)'
```

Comment:
Document the default here somehow. It might be surprising to people familiar with the notebook API that GET is not supported by default.